src/components/events/eventListingCard.js

```diff
@@ -51,31 +77,49 @@ const CardPrice = styled.div`
   font-weight: 600;
   padding: 5px 10px;
   border-radius: 5px;
-  font-size: 1rem;
+  font-size: 0.75rem;
+
+  ${media.tablet`
+    font-size: 1rem;
+  `};
 `
 
 const CardHeading = styled.h2`
   margin-bottom: 0;
-  line-height: 1.21;
+  line-height: 1.25;
+  font-size: 1rem;
   color: black;
+
+  ${media.tablet`
+    line-height: 1.21;
+    font-size: 1.5rem;
+  `};
 `
 
 export const EventListingCard = props => {
   const { event } = props
 
   return (
     <Card to={`/events/${event.id}`}>
-      <CardImage
+      <CardImageWrapper
         src={`${event.eventsListPicture.file.url}?fit=fill&w=400&h=225&f=faces`}
-        alt={event.eventsListPicture.title}
-        width="400"
-        height="225"
-      />
+      >
+        <CardImage
```

Comment:
Seems like you're displaying the image twice, once as an `img` and once as the background on a `div`? Is there a reason for that?

Reply:
Yes this is an old front-end trick. You use the background image as it gives you better control of the positioning and sizing, as older browsers won't support the css property object-fit.

You also include an IMG element for Accessbility (so screen readers can still interpret the img element and alt tags), but also for SEO as search engines look for img elements rather than background images.